nav-bar.js

```diff
@@ -0,0 +1,16 @@
+const mainMenu = document.querySelector('.nav-list');
+const openMenu = document.querySelector('.openMenu');
+const closeMenu = document.querySelector('.closeMenu');
+
+function show() {
+  mainMenu.classList.add('visible');
+  openMenu.style.display = 'none';
+}
+
+function close() {
+  mainMenu.classList.remove('visible');
+  openMenu.style.display = 'flex';
+}
```

Comment:
- [ ] Kindly keep in mind that this `close` function should also be invoked when the user clicks on any of the item links.
Please make the corresponding changes to do so 👍